### ISSUE TITLE
The training details about Unet parameters frozen (and no script for train sd1.x)

### ISSUE THREAD

**Issue by hda-xian** [huggingface/diffusers#8451]
### Describe the bug

Why are the UNet parameters frozen during training for SD1v5, but not for SDXL? the haggingface training sdxl script sets " Unet.train() "

### Reproduction

diffusers : 0.28
***** huggingface train sdxl *********
vae.requires_grad_(False)
text_encoder_one.requires_grad_(False)
text_encoder_two.requires_grad_(False)
t2iadapter.train()
unet.train()

*******Tencent ARC train sd1v5 **************
model.cuda()
model.eval() # model is contain all models vae ,cliptext
return model

*******Tencent ARC train sdxl **************
vae.requires_grad_(False)
text_encoder_one.requires_grad_(False)
text_encoder_two.requires_grad_(False) -> the Unet does not set no grad means Unet need grad

### System Info

Ubuntu

**Comment by sayakpaul** [comment]
Can you point me to the line of code you're referring to?

**Comment by hda-xian** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by hda-xian** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by hda-xian** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by sayakpaul** [comment]
This is done to create the proper gradient chain for the T2I adapter module.
